59v2.py

```diff
@@ -0,0 +1,155 @@
+import pandas as pd
+import os
+from person import Person
+from student import Student
+from employee import Employee
+
+def saveNewEntry(entries, ids_list, sum_of_ages):
+    name = input("Please enter a name: ")
+    id_number = getNumberOnly("Please enter the ID of the name (numbers only): ")
+
+    if id_number in entries:
+        print("ID " + id_number + " already exists.")
+        return entries, ids_list, sum_of_ages
+
+    age = int(getNumberOnly("Please enter the age: "))
+    person_types = [Student, Employee, Person]
+
+    while True:
+        person_type = input("Is this person a (0) Student, (1) Employee, or (2) Neither? ")

```

Comment:
אתה יכול לרוץ כאן בלולאה על הרשימה
for i, person_type in person_types:
ואז בכל איטרציה להדפיס את i, ואת השם של הטיפוס לדעתי אפשר ככה למצוא את השם בצורה אוטומטית:

Reply:
`person_type___name__`